noise.js

```diff
@@ -1,3 +1,23 @@
+function play(event) {
+  var note = $(this).attr("id");
+  document.getElementById(note + "Audio").play();
+}
+
 $(document).ready( function() {
-  // your code here
+  $("body").keydown(function(event) {
+    var key = document.getElementById(event.key + "Audio");
```

Comment:
Better not to mix regular JavaScript selections with jQuery, use one or the other.